static-site/app/blog/page.tsx

```diff
@@ -0,0 +1,17 @@
+import { redirect } from "next/navigation";
+
+import { getBlogPosts } from "./utils";
+
+export default function Index(): void {
+  const mostRecentPost = getBlogPosts()[0];
+
+  // _technically_ getBlogPosts() could return an empty array and then
+  // mostRecentPost would be undefined -- to make the type checker
+  // happy, if for some reason mostRecentPost is undefined, we will
+  // detect that and redirect to the 404 page
+  const redirectTo = mostRecentPost
+    ? `/blog/${mostRecentPost.blogUrlName}`
+    : `/404`;
+
+  redirect(redirectTo);
```

Comment:
yeah, i see that. 

seems to be because the image URLs are relative to `/blog` (i.e., `img/foo.png`, not `/blog/img/foo.png`), maybe? The images are showing as 404 and the HTTP request was `GET /img/oropouche_host_view.png`. 

oddly, it works fine on localhost... I will dig into the `redirect()` API and see if I need to do something different, but an initial fix might be tweaking the image URLs in the Oropouche post.

Reply:
So, yeah, this is because of how the `redirect()` API works, apparently. 

I think I'm going to take this as the push from the universe to move ahead with converting the front page to be the five most recent blog posts, which will require some additional refactoring in this PR.

Reply:
In the end, because of how Next.JS generates URLs without trailing slashes, I had to update the blog image URLs; there was no other way to resolve this (not without other knock-on damage that would have been worse). 

So the "multiple pages on the front page" thing will get kicked down the road a bit, and this PR is once again ready to be reviewed.

Note that I also added a section to the static-site README with some minimal info about blog post creation.